test-util/src/main/java/io/zeebe/test/util/asserts/TopologyAssert.java

```diff
@@ -30,7 +32,7 @@ public final TopologyAssert isComplete(final int clusterSize, final int partitio
     final List<BrokerInfo> brokers = actual.getBrokers();
 
     if (brokers.size() != clusterSize) {
-      failWithMessage("Expected broker count to be <%s> but was <%s>", clusterSize, brokers.size());
+      throw failure("Expected broker count to be <%s> but was <%s>", clusterSize, brokers.size());
```

Comment:
The javadoc from `failWithMessage` actually recommends using `throw failure` instead, as the compiler can now realize we're throwing an error (whereas with `failWithMessage` it thinks execution will continue).